src/components/Activities/Activity1/TextBubble.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import LargeTextBubbleSvg from '../../../assets/activity1/text_bubbles/large_text_bubble.svg';
+import MediumTextBubbleSvg from '../../../assets/activity1/text_bubbles/medium_text_bubble.svg';
+import SmallLeftTextBubbleSvg from '../../../assets/activity1/text_bubbles/small_left_text_bubble.svg';
+import SmallRightTextBubbleSvg from '../../../assets/activity1/text_bubbles/small_right_text_bubble.svg';
+import ExtraLargeTextBubbleSvg from '../../../assets/activity1/text_bubbles/xl_text_bubble.svg';
+
+import { TextBubbleStyles } from '../../shared/PlaynetConstants';
+
+import '../../styles/TextBubble.scss';
+
+interface TextBubbleProps {
+  text?: string;
+  contentSvg?: string;   //if broken change back to any
+  textBubbleStyle: TextBubbleStyles,
+}
+function TextBubble(props: TextBubbleProps): JSX.Element {
+  // processSvg ()
+  return (
+    <div className='text-bubble'>
+      {/* <img src = {} /> */}
+      {(props.textBubbleStyle === TextBubbleStyles.EXTRA_LARGE) ? <img src={ExtraLargeTextBubbleSvg} />: <span></span>}
+      {(props.textBubbleStyle === TextBubbleStyles.LARGE) ? <img src={LargeTextBubbleSvg} />: <span></span>}
+      {(props.textBubbleStyle === TextBubbleStyles.MEDIUM) ? <img src={MediumTextBubbleSvg} />: <span></span>}
+      {(props.textBubbleStyle === TextBubbleStyles.SMALL_LEFT) ? <img src={SmallLeftTextBubbleSvg} />: <span></span>}
+      {(props.textBubbleStyle === TextBubbleStyles.SMALL_RIGHT) ? <img src={SmallRightTextBubbleSvg} />: <span></span>}
```

Comment:
try to minimize divs/brs/spans in final result unless you have a reason for them
```suggestion
  let image;
  image = (props.textBubbleStyle === TextBubbleStyles.EXTRA_LARGE) ? ExtraLargeTextBubbleSvg : image;
  image = (props.textBubbleStyle === TextBubbleStyles.LARGE) ? LargeTextBubbleSvg : image;
  image = (props.textBubbleStyle === TextBubbleStyles.MEDIUM) ? MediumTextBubbleSvg : image;
  image = (props.textBubbleStyle === TextBubbleStyles.SMALL_LEFT) ? SmallLeftTextBubbleSvg : image;
  image = (props.textBubbleStyle === TextBubbleStyles.SMALL_RIGHT) ? SmallRightTextBubbleSvg : image;

  return (
    <div className='text-bubble'>
      {(!image) ? <></> : <img src={image} />}
```

Reply:
also, we are a bit far along so i don't think we need to go back and change it, but take a look at [this commit,](https://github.com/uclaacm/Playnet/commit/346638eb72dc4fe1de5e0e34093fec9f7bc49051) it uses id to specify background image rather than doing all these ternary operations

Reply:
I would recommend the div method because it becomes a lot more readable!

Reply:
@BryanPan342 div method? as in the original? or

Reply:
like using a div element and having the background image be associated with an id!

Reply:
ya sounds good to me! in fact, this would remove the funny delayed rendering thing you had @darren2hang if you specify width and height for each image! hmu if you have more questions :)

Reply:
ok, i changed it to use background id.